src/components/ChatWindowHeader/ChatWindowHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import CloseChatWindowButton from '../Buttons/CloseChatWindowButton/CloseChatWindowButton';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    container: {
+      height: `${theme.chatWindowHeaderHeight}px`,
+      background: '#F4F4F6',
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      paddingLeft: '1em',
+      paddingRight: '1em',
```

Comment:
Looks like the designs have a border on the bottom of this header.

Reply:
Nit - equivalent would be: `padding: 0 1em`

Reply:
made update to padding. i used ` boxShadow` as opposed to `borderBottom` as i think this looks closer to the designs, but let me know what you think!